index.html

```diff
@@ -482,29 +482,25 @@ <h2 class="form-heading">
         </div>
         <div class="form-container">
           <form
-            action="https://formspree.io/f/xqknnwvy"
+            action="https://formsubmit.co/echchafykyotmane1@gmail.com"
+            method="POST"
             id="contact_me"
-            method="post"
             class="form"
           >
             <div class="full-name">
               <input
-                class="form-input fullname"
```

Comment:
Currently, the form is not submitting as it seems to be inactive currently and it shows the following message on submitting the message! 🔴 Please consider reviewing the problem to meet the project requirements.

![image](https://user-images.githubusercontent.com/94223367/232027364-13f105f0-7443-418b-9b5b-71e8d69abbbe.png)